src/daft-dsl/src/python.rs

```diff
@@ -628,3 +633,121 @@ impl From<&PyExpr> for crate::ExprRef {
         item.expr.clone()
     }
 }
+
+impl<'py> IntoPyObject<'py> for LiteralValue {
+    type Target = PyAny;
+
+    type Output = Bound<'py, Self::Target>;
+
+    type Error = PyErr;
+
+    fn into_pyobject(self, py: Python<'py>) -> Result<Self::Output, Self::Error> {
+        fn div_rem(l: i64, r: i64) -> (i64, i64) {
+            (l / r, l % r)
+        }
+
```

Comment:
Where did this come from? I think I just implemented the same thing a few days ago, so let's consolidate so we don't have two (possibly different) conversion mechanisms.

https://github.com/Eventual-Inc/Daft/blob/main/src/daft-dsl/src/visitor.rs#L333-L391

Reply:
Thanks, could you also update the note in the visitor too? You might be able to re-use the macro too

Reply:
Updated